src/App/Pages/RecipesPerCategoryList.elm

```diff
@@ -2,19 +2,19 @@ module App.Pages.RecipesPerCategoryList exposing (view)
 
 import App.Model exposing (..)
 import Html exposing (..)
-import RemoteData exposing (RemoteData(..))
+import RemoteData exposing (RemoteData(..), WebData)
 import App.View.Components exposing (viewRemoteData)
```

Comment:
nitpick, but I find `Components` to be such a loaded word, I prefer to avoid it. We usually stick stuff under e.g. `Utils.Html`, `Utils.Json`, etc'